app/presenters/organisations/featured_news_presenter.rb

```diff
@@ -0,0 +1,48 @@
+module Organisations
+  class FeaturedNewsPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def first_featured_news
+      main_story = featured_news([org.ordered_featured_documents.first])
+      main_story = main_story[0]
+      main_story[:large] = true
+      main_story
+    end
+
+    def remaining_featured_news
+      return featured_news(org.ordered_featured_documents[1, 100]) if org.is_news_organisation?
+      all_featured_news
+    end
+
+    def all_featured_news
+      featured_news(org.ordered_featured_documents)
+    end
+
+  private
+
+    def featured_news(featured)
+      news_stories = []
+      featured.each do |news|
+        human_date = Date.parse(news["public_updated_at"]).strftime("%-d %B %Y") if news["public_updated_at"]
+        divider = " — " if human_date
```

Comment:
I think this if statement should be checking the presence of document_type instead/too? At the moment, we're getting things like this:

<img width="329" alt="screen shot 2018-06-13 at 09 46 05" src="https://user-images.githubusercontent.com/29889908/41342609-917bce54-6ef4-11e8-8031-e4ec402b1078.png">